apps/judicial-system/backend/src/app/modules/police/police.service.ts

```diff
@@ -58,7 +58,7 @@ export enum PoliceDocumentType {
   RVMV = 'RVMV', // Viðbótargögn verjanda
   RVVS = 'RVVS', // Viðbótargögn sækjanda
   RVFK = 'RVFK', // Fyrirkall
-  RVBD = 'RVBD', // Birtingarvottorð dóms - TODO: Not supported by RLS?
+  RVBD = 'RVBD', // Birtingarvottorð dóms
```

Comment:
Waiting for this to be implemented in the police system ([slack](https://islandis.slack.com/archives/C044207GJ74/p1759158014056429?thread_ts=1757672829.791389&cid=C044207GJ74))